examples/Planck/Makefile

```diff
@@ -0,0 +1,55 @@
+# This stub makefile for a Kaleidoscope plugin pulls in 
+# all targets from the Kaleidoscope-Plugin library
```

Comment:
This is a problem in a few existing snippets: the comment here is a bit outdated. There is no `Kaleidoscope-Plugin` library anymore. Could we make the comment correct and up-to-date here, so it can be copied to the other places later?